triliovault/templates/daemonset-datamover.yaml

```diff
@@ -68,21 +68,21 @@ spec:
             - name: trilio-mounts
               mountPath: /var/triliovault-mounts
             - name: triliovault-etc
-              mountPath: /etc/triliovault-datamover/triliovault-datamover.conf
+              mountPath: /etc/triliovault/triliovault-datamover.conf
               subPath: datamover.conf
               readOnly: true
-            - mountPath: /etc/triliovault-object-store.conf
+            - mountPath: /etc/triliovault/triliovault-object-store.conf
               name: triliovault-etc
               readOnly: true
               subPath: tvostore.conf
-            - mountPath: /etc/fuse.conf
+            - mountPath: /etc/triliovault/fuse.conf
```

Comment:
This is fuse conf file and should stay in /etc/ directory.

Reply:
Okay, I will change it.